.travis.yml

```diff
@@ -187,6 +187,15 @@ jobs:
                           - docker run  -i -t -v `pwd`:/odp
                               -e CC="${CC}"
                               ${DOCKER_NAMESPACE}/travis-odp-lng-ubuntu_16.04 /odp/scripts/ci/build_${ARCH}.sh
+                - stage: test 
+                  env: ARCH=x86_64 CC=clang UBUNTU_VERS="18.04"
+                  install:
+                          - true
+                  script:
+                          - if [ -z "${DOCKER_NAMESPACE}" ] ; then export DOCKER_NAMESPACE="opendataplane"; fi
+                          - docker run  -i -t -v `pwd`:/odp
+                              -e CC="${CC}"
+                              ${DOCKER_NAMESPACE}/travis-odp-lng-ubuntu_${UBUNTU_VERS} /odp/scripts/ci/build_${ARCH}.sh
```

Comment:
What about just putting it into plain environment matrix? With UBUNTU_VERSION being environment entry?

Reply:
hm, let me do a try...